src/Microsoft.AspNet.Hosting/IWebApplication.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Http.Features;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNet.Hosting
+{
+    /// <summary>
+    /// Represents a configured web application
+    /// </summary>
+    public interface IWebApplication
+    {
+        /// <summary>
+        /// The <see cref="IFeatureCollection"/> exposed by the configured server.
+        /// </summary>
+        IFeatureCollection ServerFeatures { get; }
```

Comment:
Is Server necessary as a prefix, or is it sufficiently implied, i.e. the interface isn't called IServerFeatureCollection...